server/model/monitor.js

```diff
@@ -595,7 +595,7 @@ class Monitor extends BeanModel {
                         let data = res.data;
 
                         // convert data to object
-                        if (typeof data === "string") {
+                        if (typeof data === "string" && res.headers["content-type"] !== "application/json") {
```

Comment:
I am unsure if this is logic is correct:
So if I send out `{"a":"b"}` as `application/json` it won't be parsed.

Reply:
If the response body is a valid JSON object, it would have been deserialized as a plain JavaScript object after passing through axios.

This part of code only handle edge cases where the body is not deserialized by default.